### DISCUSSION TITLE
WinNUT 2.2 or 2.3 ??

### DISCUSSION THREAD

**Discussion by davidemiccone** [nutdotnet/WinNUT-Client#181] [Q&A]
I'm a bit confused.

On the repo I found the branche dev-2.3 but if I download this Branch, compile and run it show "Version 1.0.0.0".

Is this a Bug?

Or maybe I have to download Branch dev-2.2 ?

Thank you for suggestions

**Comment by gbakeman** [comment]
Hi @davidemiccone,

You were right originally, it's the Dev-2.3 branch that you want. I've been changing how WinNUT is deployed, so the version is set based on the tag when it's time to deploy. I think there was some reason that I had to leave a version property defined, but I agree that it's not the best solution. Maybe I should manually update the version number in the project definition files along with tags.
